HelloTorchIOS/HelloTorchIOS/ViewController.swift

```diff
@@ -10,16 +10,9 @@ import UIKit
 
 class ViewController: UIViewController {
 
-    override func viewDidLoad() {
-        super.viewDidLoad()
-        // Do any additional setup after loading the view, typically from a nib.
+    let strobeLights = StrobeLights()
+    
+    @IBAction func toggleTorch(sender: UIButton) {
+        strobeLights.toggleStrobe()
```

Comment:
Since `toggleStrobe` has all the logic you could here directly call `incrementCounter` since is the method you call from timer. It would make more sense to have only one method access to the logic from outside instead of two.

Reply:
I don't understand :/